src/allmydata/frontends/magic_folder.py

```diff
@@ -153,21 +258,38 @@ def startService(self):
         self._stats_provider.count('magic_folder.dirs_monitored', 1)
```

Comment:
The call to `_scan` needs to be after we are ready, since `_scan` now looks at the Tahoe filesystem.